### ISSUE TITLE
Create "intermediate" scenario

### ISSUE THREAD

**Issue by docktermj** [Senzing/docker-compose-aws-ecscli-demo#9]
The intermediate scenario adds the following:

1. Use of AWS Fargate
1. Use of Elastic File System
1. Example of connecting to Aurora
1. Example of connecting to AWS SQS
1. Example of scaling up

**Comment by docktermj** [comment]
Interesting little tidbit on FARGATE versions:

https://stackoverflow.com/questions/61351000/mounting-efs-to-aws-fargate-using-terraform-platformtaskdefinitionincompatibil

Turns out EFS isn't available on FARGATE 1.3.0

**Comment by docktermj** [comment]
On https://aws.amazon.com/blogs/containers/aws-fargate-platform-versions-primer/

> These users could either start experimenting with 1.4.0 by explicitly deploying to it or wait until we move the tag LATEST to point to platform version 1.4.0. We plan to make this change at some point in Q3/2020, thus giving customers a few months to test 1.4.0 before it gets tagged as LATEST. Expect a formal announcement for this change.

So currently we have to find a way to force 1.4.0 using `ecs-cli`.

**Comment by docktermj** [comment]
Ran https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-fargate.html demonstration.  Looked at https://console.aws.amazon.com/ecs/home?#/clusters > [cluster] > "Tasks" tab > [task] and saw:

```
 Launch type   FARGATE
Platform version   1.3.0
```
So FARGATE 1.3.0 is deployed by the `ecs-cli`  version 1.19.1 (a20cf74)

**Comment by docktermj** [comment]
Looks like `ecs-cli` is being worked on:

1. https://github.com/aws/amazon-ecs-cli/pull/1019
2. https://github.com/aws/amazon-ecs-cli/pull/1056

**Comment by docktermj** [comment]
Currently progress on AWS Elastic File System (EFS) blocked by https://github.com/aws/amazon-ecs-cli/issues/1043

Am making progress on other fronts to enable SQS:

1. https://github.com/Senzing/stream-producer/issues/4
2. https://github.com/Senzing/stream-loader/issues/82

**Comment by docktermj** [comment]
Visited https://github.com/aws/amazon-ecs-cli/issues/1043.   A pull request was made; so things are progressing.

**Comment by docktermj** [comment]
https://github.com/aws/amazon-ecs-cli/issues/1043 still open (i.e. not fixed)

**Comment by docktermj** [comment]
https://github.com/aws/amazon-ecs-cli/issues/1043 still open (i.e. not fixed)

**Comment by docktermj** [comment]
https://github.com/aws/amazon-ecs-cli/issues/1043 unblocked yesterday!  Will need to download version 1.20.0

**Comment by docktermj** [comment]
New issue found and filed at https://github.com/aws/amazon-ecs-cli/issues/1083

**Comment by docktermj** [comment]
At https://github.com/aws/amazon-ecs-cli/issues/1083#issuecomment-656800284  There is a link to a new offering at https://aws.amazon.com/blogs/containers/aws-and-docker-collaborate-to-simplify-the-developer-experience/  which is a tighter integration of Docker-compose and AWS Fargate.

**Comment by docktermj** [comment]
Posted an issue for `ecs-cli`:
https://github.com/aws/amazon-ecs-cli/issues/1083#issuecomment-656922345

**Comment by docktermj** [comment]
Response posted last week at https://github.com/aws/amazon-ecs-cli/issues/1083#issuecomment-657714131.  Will be reviewing/trying it.

**Comment by docktermj** [comment]
Pull requests:

1. https://github.com/Senzing/docker-compose-aws-ecscli-demo/pull/13
2. https://github.com/Senzing/docker-compose-aws-ecscli-demo/pull/23

**Comment by docktermj** [comment]
https://github.com/aws/amazon-ecs-cli/issues/1083
